src/components/PlayerUI.tsx

```diff
@@ -0,0 +1,30 @@
+import { useState, useEffect} from 'react';
+import WebPlayback from './WebPlayBack';
+
+//Acceptable format for props data.
+export interface IPlayerUIProps {
+    trackName: string,
+    albumName: string,
+    artistName: string
+}
+//@ts-ignore
+function PlayerUI(props: IPlayerUIProps) {
+
+    //Deconstruction props
+    const {
+        trackName,
+        albumName,
+        artistName
+    } = props;
+
+    //Displaying to the user props.
+    return(
+        <div> 
+            <div>Name: {trackName}</div>
```

Comment:
I like this component! It's very simple and can be reused easily to display information

Reply:
Thanks! Could you give an example? I might not have taken my medicine so I can't see it. How?